### ISSUE TITLE
build of 1.2.4 on windows 8 fails - cairo.h not found (all good with 1.2.3)

### ISSUE THREAD

**Issue by timknip** [Automattic/node-canvas#591]
[commit 86d6ec59b7cb0e4ffce47b9cee81a9ce20afcb1e](https://github.com/Automattic/node-canvas/commit/86d6ec59b7cb0e4ffce47b9cee81a9ce20afcb1e) seems to be the culprit.

In GTK 2 ```cairo.h``` lives in ```include/cairo/cairo.h```
Hence its not found and the build fails

EDIT: Guess this PR will fix the issue https://github.com/Automattic/node-canvas/pull/589

**Comment by LinusU** [comment]
@timknip I'm rather sure that we'll need #589 to fix this, but do you think that you have time to try with `1.2.5`? `1.2.4` was broken and didn't work at all.

**Comment by timknip** [comment]
@LinusU 1.2.5 fails too... so yeah, https://github.com/Automattic/node-canvas/pull/589 for sure is needed.

```
Canvas.h(19): fatal error C1083: Cannot open include file: 'cairo.h': No such file or directory 
```

**Comment by timknip** [comment]
@LinusU Let me know if you merged the PR... btw: ```canvas-browserify``` uses ```<1.2.4``` for now to prevent install problems. As soon as ```canvas > 1.2.3``` builds again, i'll of course update.

**Comment by timknip** [comment]
@LinusU Tested 1.2.5 on win8 (x64) with https://github.com/Automattic/node-canvas/pull/589 merged. 
Seems to work fine. Tests are passing.
```node test/server.js``` is also looking good

**Comment by LinusU** [comment]
Fixed in `1.2.6` :+1:
